src/mutation.cpp

```diff
@@ -479,6 +481,11 @@ void player::activate_mutation( const trait_id &mut )
         print_health();
         tdata.powered = false;
         return;
+    } else if( mut == trait_PREENING ) {
+        add_morale( MORALE_PREENED, 5, 30, 30_minutes, 24_minutes );
+        add_msg_if_player( m_good, _( "You preen yourself, straightening out all those ruffled feathers." ) );
```

Comment:
This seems infinitely repeatable, so it should have a time cost to counteract that. `moves -= 200` should be more than enough.